tools/cldr-code/src/main/java/org/unicode/cldr/util/Organization.java

```diff
@@ -56,6 +55,7 @@ public enum Organization {
     special("High Coverage and Generated"),
     srilanka("Sri Lanka ICTA", "Sri Lanka"),
     surveytool("Survey Tool"),
+    unaffiliated("Unaffiliated", "Guest"),
```

Comment:
this is added -- preferred name "Unafilliated"
the variant name "Guest" facilitates the db conversion when this code runs the first time, and could be dropped later; maybe all the variant names could be dropped; once the db is normalized they might not serve any purpose

Reply:
they would serve for comparison with earlier data sets

Reply:
i would keep them because they give a key to the organization identity